.github/workflows/docker-publish-github.yml

```diff
@@ -1,45 +0,0 @@
-name: Publish OSML Models GitHub Container
-
-on:
-  workflow_call:
-
-env:
-  REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
-
-jobs:
-  build-and-push-image:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
-    steps:
-      - name: Free up the build space
-        uses: easimon/maximize-build-space@master
-        with:
-          root-reserve-mb: 30000
-          swap-size-mb: 1024
-          remove-dotnet: 'true'
-          remove-android: 'true'
-          remove-codeql: 'true'
-          remove-haskell: 'true'
```

Comment:
Could you move this block of code to the `docker-publish` workflow? This is needed since the docker container is so big, we are required to remove any unnecessary packages in github runner so we can build the docker container.

Reply:
done